substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/phases/NativeImageInlineDuringParsingPlugin.java

```diff
@@ -348,84 +332,48 @@ InvocationResult analyzeMethod(GraphBuilderContext b, AnalysisMethod method, Cal
             }
```

Comment:
NativeImageInlineDuringParsingPlugin is removed when making graphbuilder for callee